packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -223,139 +223,146 @@ export function SchemaDrivenAnnotationEditor(
-              <Alert severity="info">
-                <b>{entityJson.name}</b> has no annotations. Click the{' '}
-                <AddToList /> button to annotate.
-              </Alert>
-            )}
-          <Form
-            validator={validator}
-            className="AnnotationEditorForm"
-            liveValidate={liveValidate}
-            noHtml5Validate={true}
-            experimental_defaultFormStateBehavior={{
-              emptyObjectFields: 'skipDefaults',
-            }}
-            fields={{
-              ObjectField: CustomObjectField,
-            }}
-            templates={{
-              ArrayFieldDescriptionTemplate: ArrayFieldDescriptionTemplate,
-              ArrayFieldItemTemplate: ArrayFieldItemTemplate,
-              ArrayFieldTemplate: ArrayFieldTemplate,
-              ArrayFieldTitleTemplate: ArrayFieldTitleTemplate,
-              BaseInputTemplate: BaseInputTemplate,
-              FieldErrorTemplate: FieldErrorTemplate,
-              FieldTemplate: FieldTemplate,
-              ObjectFieldTemplate: ObjectFieldTemplate,
-              WrapIfAdditionalTemplate: WrapIfAdditionalTemplate,
-              ButtonTemplates: ButtonTemplate,
-              DescriptionFieldTemplate: DescriptionFieldTemplate,
-              /* Errors are displayed by an Alert component below, so we don't show the builtin ErrorList */
-              ErrorListTemplate: () => null,
-            }}
-            ref={ref}
-            disabled={mutation.isLoading}
-            schema={
-              {
-                ...(validationSchema ?? {}),
-                patternProperties: {
-                  ...(validationSchema?.patternProperties ?? {}),
-                  ...patternPropertiesBannedKeys,
-                },
-                additionalProperties:
-                  validationSchema?.additionalProperties ?? true,
-              } as JSONSchema7
-            }
-            uiSchema={{
-              'ui:options': {
-                copyable: true,
-                duplicateKeySuffixSeparator: '_',
-              },
-              additionalProperties: {
-                'ui:field': AdditionalPropertiesSchemaField,
-              },
-            }}
-            transformErrors={transformErrors}
-            formData={formData}
-            onChange={({ formData }) => {
-              if (onChange) {
-                onChange(formData)
+          {entityJson && isEmpty(formData) && schema === null && (
+            <Alert severity="info">
+              <Box display={'flex'} alignItems={'center'} gap={0.5}>
+                <Typography variant={'smallText1'}>
+                  <b>{entityJson.name}</b> has no annotations. Click the{' '}
+                </Typography>
+                <AddToList />
+                <Typography variant={'smallText1'}>
+                  button to annotate.
+                </Typography>
+              </Box>
+            </Alert>
+          )}
+          {formData != undefined && (
```

Comment:
Added this check to not render if `undefined`. Not a problem for entities with no annotations, `formData` will be an empty object in that case. No other changes in this file--the remaining diff is from adding this conditional wrapper